kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/status/modem/ModemInterfaceStatus.java

```diff
@@ -147,6 +149,13 @@ public Boolean isGpsSupported() {
         return this.gpsSupported;
     }
 
+    /*
+     * @since 2.8
```

Comment:
Have we upticked the api version to 3.0, isn't it? :confused:

Reply:
I think the motivation behind this is given by the fact that this PR will be backported to 5.6 and will first be available there. I'm simply relying on the previous work done by @sfiorani under the suggestion of @nicolatimeus for #5381

Feel free to correct me on this obviously.

Reply:
If this code will be contributed to 5.6, the 2.8 is correct.